### ISSUE TITLE
"wlo1" not automatically detected => "ERROR interface not found, sheeplessly quiting". Includes workaround

### ISSUE THREAD

**Issue by sanderjo** [cvmiller/v6disc#4]
The command `./v6disc.sh` works when my laptop is connected via wired (via interface enx1cbfce2fc36d)
The same command does not work when only my wifi is connected:

```
sander@zwart2204:~/git/v6disc$ ./v6disc.sh 
-- Searching for interface(s) 
ERROR interface not found, sheeplessly quiting

```

Cause seems to be in the filtering out 'lo.?'

`intf_list=$(ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v 'lo.?:' |grep -v -i no-carrier | cut -d ":" -f 2 | cut -d "@" -f 1 )`
https://github.com/cvmiller/v6disc/blob/master/v6disc.sh#L376C2-L377C1

My guess is that `grep -E -v 'lo.?:'` wants to filter out loopback `lo:`, but also filters out `wlo1`.

**My workaround: a space before the `lo` and it's working**
```
	intf_list=$(ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v ' lo.?:' |grep -v -i no-carrier | cut -d ":" -f 2 | cut -d "@" -f 1 )
```
result
```
$ ./v6disc.sh 
-- Searching for interface(s) 
-- Found interface(s):  wlo1 
-- INT:wlo1	prefixs: 2001:dd3c:4300:200 
-- Detecting hosts on wlo1 link 
```

**Debugging was**

```
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)'
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
2: wlo1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DORMANT group default qlen 1000
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default 
sander@zwart2204:~/git/v6disc$ 
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v 'lo.?:' |grep -v -i no-carrier | cut -d ":" -f 2 | cut -d "@" -f 1
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v 'lo.?:' |grep -v -i no-carrier 
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v 'lo.?:' 
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default 
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)' 
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
2: wlo1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DORMANT group default qlen 1000
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default 
sander@zwart2204:~/git/v6disc$ 
sander@zwart2204:~/git/v6disc$ 
sander@zwart2204:~/git/v6disc$ ip link | grep -E -i '(state up|multicast,up|up,)' | grep -E -v ' lo.?:' |grep -v -i no-carrier | cut -d ":" -f 2 | cut -d "@" -f 1
 wlo1
```

**Comment by cvmiller** [comment]
I agree with your fix, and will update vdisc soon. Thanks for pointing this out.

**Comment by cvmiller** [comment]
Fixed in v2.3.5
